core/whistle_media-1.0.0/src/wh_media_map.erl

```diff
@@ -362,7 +362,10 @@ get_map(?WH_MEDIA_DB = Db, PromptId) ->
             #media_map{id=MapId
                        ,account_id=Db
                        ,prompt_id=PromptId
-                       ,languages=wh_json:new()
+                       ,languages=wh_json:set_value(wh_media_util:default_prompt_language()
+                                                    ,<<"/", ?WH_MEDIA_DB/binary, "/", (wh_media_util:default_prompt_language())/binary, "%2F", PromptId/binary>>
```

Comment:
It's OK with 
(whistle_apps@kz527.onnet.su)1> wh_media_util:get_prompt(<<"vm-saved">>).
<<"prompt://system_media/vm-saved/en-us">>
(whistle_apps@kz527.onnet.su)2> 

I think we need to start from the point, that prompts could be found only when they are in cache.
It it is absent Kazoo fails to build path